inc/namespace.php

```diff
@@ -39,7 +39,7 @@ function enqueue_assets() {
 		 *
 		 * @var array An array of default categories to consent to automatically.
 		 */
-		'alwaysAllowCategories' => apply_filters( 'altis.consent.always_allow_categories', [ 'functional', 'statistics-anonymous' ] ),
+		'alwaysAllowCategories' => apply_filters( 'altis.consent.allowlisted_categories', [ 'functional', 'statistics-anonymous' ] ),
```

Comment:
I'm really sure what this change is fixing - the original name seems to match the intention of the setting more clearly.

Reply:
I suppose I could swap them and use `always_allow_categories`... I just realized I was doing the same thing twice in two different ways and wanted to unify them.